src/node_file.cc

```diff
@@ -992,6 +993,23 @@ void Close(const FunctionCallbackInfo<Value>& args) {
   }
 }
 
+static void FastClose(Local<Object> recv,
+                      const int32_t fd,
+                      // NOLINTNEXTLINE(runtime/references) This is V8 api.
+                      v8::FastApiCallbackOptions& options) {
+  Environment* env = Environment::GetCurrent(recv->GetCreationContextChecked());
+
+  env->RemoveUnmanagedFd(fd);
```

Comment:
If `fallback` is true, this will be called twice, right? If so, is it safe to be called twice?

Reply:
Fixed it.